packages/notifications/push/amplify_push_notifications_pinpoint/example/lib/main.dart

```diff
@@ -86,6 +68,30 @@ class _MyAppState extends State<MyApp> {
     });
   }
 
+  void bgHandler(PushNotificationMessage pushNotificationMessage) async {
+    print('bgHandler called');
+    setState(() {
+      backgroundMessage = pushNotificationMessage;
+    });
+    // try {
+    //   WidgetsFlutterBinding.ensureInitialized();
+    //   print('BG handler invoked');
+    //   final prefs = await SharedPreferences.getInstance();
+    //   await prefs.reload();
+    //   var globalBgCallbackCount = prefs.getInt(globalBgCallbackKey);
+    //   globalBgCallbackCount =
+    //       globalBgCallbackCount != null ? (globalBgCallbackCount + 1) : 1;
+    //   await prefs.setInt(
+    //     globalBgCallbackKey,
+    //     globalBgCallbackCount,
+    //   );
+    //   print('globalBgCallbackCount in handler -> $globalBgCallbackCount');
+    // } on Exception catch (e) {
+    //   print(' error in handler: $e');
+    // }
+    // return;
```

Comment:
Should remove?

Reply:
needed to demonstrate working of this API in killed state. So, uncommented it.